x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/PutTrainedModelDefinitionPartAction.java

```diff
@@ -73,7 +71,7 @@ public Request(String modelId, BytesReference definition, int part, long totalDe
         public Request(StreamInput in) throws IOException {
             super(in);
             this.modelId = in.readString();
-            this.definition = in.readBytesReference();
+            this.definition = in.readString();
```

Comment:
Can the wire type of the field be changed like this between versions?

Reply:
this is going in v8. This was only ever put in v8